.github/workflows/cla.yml

```diff
@@ -0,0 +1,22 @@
+name: Contributor License Agreement (CLA)
+
+on:
+  pull_request_target:
+    types: [opened, synchronize]
+  issue_comment:
+    types: [created]
+
+jobs:
+  cla:
+    runs-on: ubuntu-latest
+    if: |
+      (github.event.issue.pull_request 
+        && !github.event.issue.pull_request.merged_at
+        && contains(github.event.comment.body, 'signed')
```

Comment:
Should this check the source of the comment? Could you break this by just writing `signed` in an arbitrary comment in a PR?

Reply:
that's just an optimization to save the billed seconds, as all comments on PRs will trigger this job, however:
* if we run the action and do the check programmatically there, it takes around 20 seconds
* here however, the action is not booted and this check takes 1-3 seconds to execute

there are more sophisticated checks inside the action code though